api/src/pcapi/core/finance/factories.py

```diff
@@ -126,6 +131,10 @@ class CollectivePricingFactory(_BasePricingFactory):
     revenue = factory.LazyAttribute(lambda pricing: int(100 * pricing.collectiveBooking.collectiveStock.price))
 
 
+class InvoicedCollectivePricingFactory(CollectivePricingFactory):
+    status = models.PricingStatus.INVOICED
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** Est-ce qu'on ferait pas mieux de faire un `CollectivePricingFactory(status=models.PrincingStatus.INVOICED)`  vu qu'il n'y a pas d'autres paramètres ? ‏